### DISCUSSION TITLE
Library for SHT10 humidity and temperature sensor, ESP32

### DISCUSSION THREAD

**Discussion by ZDBak** [micropython/micropython#10515] [Libraries & Drivers]
Hi,

is there any MicroPython library for SHT10 sensor designed for ESP32? I would like to use the sensor for my project, but I can't find one.

Thanks.

**Comment by robert-hh** [comment]
You could look at https://awesome-micropython.com. There are drivers for SHT30 and SHT31. Maybe that helps.

**Reply by robert-hh** [reply]
Looking at the data sheets, I see that t he sht10 is completely different to the sht30, not even using I2C. So the SHT30 code is not helpful. The communication itself is not time critical and not complicated, "just" some bit fiddling.
